packages/balances-react/package.json

```diff
@@ -52,12 +50,16 @@
     "@types/react": "^18.0.17",
     "eslint": "^8.4.0",
     "jest": "^29.7.0",
+    "jotai": "^2.6.4",
+    "jotai-effect": "^0.5.0",
     "react": "^18.2.0",
     "ts-jest": "^29.1.1",
     "typescript": "^5.2.2"
   },
   "peerDependencies": {
     "@polkadot/util-crypto": "12.x",
+    "jotai": "*",
```

Comment:
Not sure if this is good, we should not expect that all consuming apps will have or want to have jotai

Reply:
concur